src/glossary.js

```diff
@@ -89,21 +93,48 @@ function Glossary(terms, selectors) {
   this.addEventListener(document.body, 'keyup', this.handleKeyup.bind(this));
 }
 
-/** Populate internal list.js list of terms */
 Glossary.prototype.populate = function() {
+  var self = this;
+  var terms = this.terms.map(function(term) {
+    return {
+      'term': term.term,
```

Comment:
Not sure that we need this `map`, but in any case, prefer not to use quotes for key names in object declarations.